test/Dialect/Calyx/emit-invoke.mlir

```diff
@@ -0,0 +1,32 @@
+// RUN: circt-translate --export-calyx --split-input-file --verify-diagnostics %s | FileCheck %s --strict-whitespace
```

Comment:
@rachitnigam  @cgyurgyik For the emit invoke, you can see the `emit-invoke.mlir` test.